validator/validator.go

```diff
@@ -0,0 +1,68 @@
+package validator
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/screwdriver-cd/sd-cmd/screwdriver/api"
+	"github.com/screwdriver-cd/sd-cmd/util"
+)
+
+// Validator is a type to validate yaml.
+// It receives strings which input by a user.
+type Validator struct {
+	cmdPath    string
+	yamlString string
+	sdAPI      api.API
+}
+
+// Run is a method to validate yaml.
+func (v *Validator) Run() error {
+	validateResponse, err := v.sdAPI.ValidateCommand(v.yamlString)
+	if err != nil {
+		return fmt.Errorf("Post failed:%v", err)
+	}
+
+	if len(validateResponse.Errors) != 0 {
+		errorMessage := ""
+		for _, error := range validateResponse.Errors {
+			errorMessage += error.Message + "\n"
+		}
+		return fmt.Errorf("Command is not valid for the following reasons:\n%v", errorMessage)
+	}
+
+	fmt.Println("Validation Succesfull.")
```

Comment:
Successfull

Reply:
I think
"Validation completed successfully."
is better.

Reply:
I fixed the message.